epub33/rs/index.html

```diff
@@ -1588,25 +1588,44 @@ <h4>Initial containing block dimensions</h4>
 						<dt>XHTML</dt>
 						<dd>
 							<p>
-								<span id="confreq-fxl-rs-xhtml-icb-def"> Reading systems MUST create the initial
+								<span id="confreq-fxl-rs-xhtml-icb-def">Reading systems MUST create the initial
 									containing block (ICB) using the width and height expressions declared in the
 										<code>viewport</code>
 									<code>meta</code> tag for [=XHTML content documents=], as defined in <a
 										data-cite="epub-33#sec-fxl-icb-html">Expressing in HTML</a> [[epub-33]].</span>
-								<span id="confreq-fxl-rs-xhtml-icb" data-tests="#fxl-xhtml-icb">They MUST clip content
-									positioned outside of the ICB.</span>
+								<span id="confreq-fxl-rs-xhtml-icb" data-tests="#fxl-xhtml-icb,fxl-xhtml-icb_multi">They
+									MUST clip content positioned outside of the ICB.</span>
 							</p>
-							<p id="confreq-fxl-rs-xhtml-icb-ratio"> When the ICB aspect ratio does not match the aspect
+							<p>If the width or height values in the <code>viewport</code>
+								<code>meta</code> tag contain a non-numeric character but start with a number (e.g., the
+								value includes a unit of length declaration such as "500px"), the number prefix is used
+								as the pixel value.</p>
```

Comment:
I'm fine either way. If we say "should", does that leave the potential to use non-pixel values, though? Do we need to say something like:

> the number prefix SHOULD be used as the pixel value, otherwise the value MUST be treated as invalid

Reply:
Agreed; I have made the change.